Documentation/helm-ceph-cluster.md

```diff
@@ -32,20 +32,68 @@ Rook currently publishes builds of this chart to the `release` and `master` chan
 
 The release channel is the most recent release of Rook that is considered stable for the community.
 
+**Before installing, review the values.yaml to confirm if the default settings need to be updated.
+The [defaults](https://github.com/rook/rook/tree/{{ branchName }}/cluster/charts/rook-ceph-cluster/values.yaml)
+are only an example and not likely to apply to your cluster.**
+
+The example install assumes you have created a values-override.yaml.
+
 ```console
 helm repo add rook-release https://charts.rook.io/release
-helm install --create-namespace --namespace rook-ceph rook-ceph --set operatorNamespace=rook-ceph rook-release/rook-ceph-cluster
+helm install --create-namespace --namespace rook-ceph rook-ceph-cluster \
+   --set operatorNamespace=rook-ceph rook-release/rook-ceph-cluster -f values-override.yaml
+```
+
+## Configuration
+
+The following tables lists the configurable parameters of the rook-operator chart and their default values.
+
+| Parameter             | Description                                                          | Default     |
+| --------------------- | -------------------------------------------------------------------- | ----------- |
+| `operatorNamespace`   | Namespace of the Rook Operator                                       | `rook-ceph` |
+| `configOverride`      | Cluster ceph.conf override                                           | <empty>     |
+| `toolbox.enabled`     | Enable Ceph debugging pod deployment. See [toolbox](ceph-toolbox.md) | `false`     |
+| `toolbox.tolerations` | Toolbox tolerations                                                  | `[]`        |
+| `toolbox.affinity`    | Toolbox affinity                                                     | `{}`        |
+| `monitoring.enabled`  | Enable Prometheus integration, will also create necessary RBAC rules | `false`     |
+| `cephClusterSpec.*`   | Cluster configuration, see below                                     | See below   |
+
+
+### Ceph Cluster Spec
+
+The `CephCluster` CRD takes its spec from `cephClusterSpec.*`. This is not an exhaustive list of parameters.
+For the full list, see the [Cluster CRD](ceph-cluster-crd.md) topic.
```

Comment:
We should document that the "monitoring" section needs to be moved out of the cephClusterSpec

Reply:
Ok, added some clarification around the monitoring and other settings that should be reviewed. Is that more clear?

Reply:
Yeah, lgtm